public/app/features/teams/UserTeamsList.tsx

```diff
@@ -0,0 +1,88 @@
+import { css } from '@emotion/css';
+import React, { useEffect, useMemo } from 'react';
+import { connect, ConnectedProps } from 'react-redux';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { CellProps, InteractiveTable, Icon, Column, Avatar, useStyles2 } from '@grafana/ui';
+import { Stack } from '@grafana/ui/src/unstable';
+import { StoreState, Team } from 'app/types';
+
+import { Page } from '../../core/components/Page/Page';
+import { TableWrapper } from '../admin/Users/TableWrapper';
+
+import { loadUsersTeams } from './state/actions';
+
+type Cell<T extends keyof Team = keyof Team> = CellProps<Team, Team[T]>;
+export interface OwnProps {}
+
+export const UserTeamsList = ({ teams, hasFetched, loadUsersTeams }: Props) => {
+  const styles = useStyles2(getStyles);
+
+  useEffect(() => {
+    loadUsersTeams();
+  }, [loadUsersTeams]);
+
+  const columns: Array<Column<Team>> = useMemo(
+    () => [
+      {
+        id: 'avatarUrl',
+        header: '',
+        cell: ({ cell: { value } }: Cell<'avatarUrl'>) => value && <Avatar src={value} alt="User avatar" />,
+      },
+      {
+        id: 'name',
+        header: 'Name',
+        cell: ({ cell: { value } }: Cell<'name'>) => value,
+      },
+    ],
+    []
+  );
+
+  return (
+    <Page navId="profile/teams" subTitle="Teams that you are a member of">
+      <Page.Contents isLoading={!hasFetched}>
+        {teams.length === 0 ? (
+          <div className={styles.noTeamsWrapper}>
+            <Icon name="users-alt" size="xxl" />
+            <span>You are currently not a member of any teams.</span>
+          </div>
```

Comment:
```suggestion
          <Flex direction={'column'} alignItems={'center'} gap={1}>
            <Icon name="users-alt" size="xxl" />
            <span>You are currently not a member of any teams.</span>
          </Flex>
```

This would be a good opportunity to use our new layout components (can be imported via `import { Flex } from '@grafana/ui/src/unstable';`). This way you won't need the `getStyles` function and code related to it.